test/Microsoft.OpenApi.SmokeTests/ApiGurus.cs

```diff
@@ -0,0 +1,107 @@
+﻿using Microsoft.OpenApi.Readers;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Net;
+using System.Net.Http;
+using System.Threading.Tasks;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Microsoft.OpenApi.SmokeTests
+{
+    [Collection("DefaultSettings")]
+    public class ApisGuruTests
+    {
+        private static HttpClient _httpClient = new HttpClient(new HttpClientHandler() {
```

Comment:
If you have the static constructor, I think it's better to move the assignment to the static constructor.